### PR TITLE
Ratings & Reviews renders all real data

### PR DESCRIPTION
**Pull request by jordanabbasi**
* Fills all components with real data
* Adds styling to make reviews list scrollable and fully within viewport

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -35,13 +35,17 @@ export default function App() {
...
       .then((data) => {
         setReviewsMeta(data);
       });
-    axios.get('/reviews', { params: { product_id: productID, sort: 'relevant', count } })
+    axios.get('/reviews', {
+      params: {
+        product_id: productID, sort: 'relevant', page: 1, count: 500,
+      },
+    })
       .then((result) => result.data.results)
```
Comment (alishaedington) [inline client/src/components/App.jsx:49]: were you planning on moving this down to the component you need it?

File: client/src/components/R&R/FactorSlider.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  return (
+    <div className="rr-factor-slider-container">
+      <h4>{name}</h4>
+      <div className="rr-factor-slider">
+        <span style={{ left: `${minPosition + (percent / 100) * (maxPosition - minPosition)}rem` }}>&#9660;</span>
+      </div>
+      <div>
+        <span className="rr-factor-slider-label">Too small</span>
+        <span className="rr-factor-slider-label" style={{ float: 'right' }}>Too large</span>
```
Comment (alishaedington) [inline client/src/components/R&R/FactorSlider.jsx:18]: These will need to eventually render depending on the specific factor. styling can be moved to style sheet? if not it isn't too distracting in this file.

File: client/src/components/R&R/ProductBreakdown.jsx [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
+
+  for (const characteristic in characteristics) {
+    const { value, id } = characteristics[characteristic];
+    sliders.push(<FactorSlider name={characteristic} value={value} key={id} />);
+  }
   return (
-    <div>
+    <div className="rr-product-breakdown">
+      {sliders}
     </div>
```
Comment (alishaedington) [inline client/src/components/R&R/ProductBreakdown.jsx:15]: i'm unfamiliar with this syntax? sliders is an array of components, so will it automatically render  all the components in this array?

**Review by alishaedington — APPROVED** [review]
Great job. comments are mostly questions I had.

File: client/src/components/App.jsx (hunk 1 above)
Comment (jordanabbasi) [reply client/src/components/App.jsx:49]: Yes, will in a later PR!

File: client/src/components/R&R/FactorSlider.jsx (hunk 2 above)
Comment (jordanabbasi) [reply client/src/components/R&R/FactorSlider.jsx:18]: Ahh. Good catch - thx.  Will update for next PR.

File: client/src/components/R&R/ProductBreakdown.jsx (hunk 3 above)
Comment (jordanabbasi) [reply client/src/components/R&R/ProductBreakdown.jsx:15]: Exactly.  Just like when you do Array.map() to create an array of components.  Same idea.

**Pull request merged**